lms/djangoapps/certificates/apis/v0/views.py

```diff
@@ -120,3 +124,132 @@ def get(self, request, username, course_id):
                 "grade": user_cert.get('grade')
             }
         )
+
+
+class CertificatesListView(GenericAPIView):
+    """
+        **Use Case**
+
+            * Get the list of viewable course certificates for a specific user.
+
+        **Example Request**
+
+            GET /api/certificates/v0/certificates/{username}
+
+        **GET Parameters**
+
+            A GET request must include the following parameters.
+
+            * username: A string representation of an user's username.
+
+        **GET Response Values**
+
+            If the request for information about the user's certificates is successful,
+            an HTTP 200 "OK" response is returned.
+
+            The HTTP 200 response contains a list of dicts with the following keys/values.
+
+            * username: A string representation of an user's username passed in the request.
+
+            * course_id: A string representation of a Course ID.
+
+            * course_display_name: A string representation of the Course name.
+
+            * course_organization: A string representation of the organization associated with the Course.
+
+            * certificate_type: A string representation of the certificate type.
+                Can be honor|verified|professional
+
+            * created_date: Date/time the certificate was created, in ISO-8661 format.
+
+            * status: A string representation of the certificate status.
+
+            * is_passing: True if the certificate has a passing status, False if not.
+
+            * download_url: A string representation of the certificate url.
```

Comment:
Now that we primarily support web certificates, why is this called `download_url`?

Reply:
I wanted to keep it consistent with the existing code. We can rename it when this course certificates code gets cleaned up, refactored, and moved to the credentials service.

Reply:
hmm.. we need to determine whether new APIs that we build along the way should be more correct and modern or more consistent (with older legacy ones).